includes/tracks/class-wc-site-tracking.php

```diff
@@ -134,4 +156,48 @@ public static function init() {
 			}
 		}
 	}
+
+	/**
+	 * Set up frontend tracking.
+	 */
+	public static function init_frontend_events() {
```

Comment:
If we add an action to this function, plugins would be able to hook into this if they need to track front end stuff.

Reply:
That's a great idea, I'll add that now. We'd likely use this this in WooCommerce Blocks plugin.

Reply:
Added here: 17ec0809fb6e1ff5b8d12128f71d02025e457fa5